src/FastRoute.php

```diff
@@ -475,9 +315,10 @@ static function ($matched, Route $route) use ($path, $method) {
      */
     private function injectItems(): void
     {
-        foreach ($this->items as $index => $item) {
-            $this->injectItem($item);
-            unset($this->items[$index]);
+        if ($this->routes === []) {
```

Comment:
What's the purpose of the check? Why would method be called more than once?

Reply:
Roadrunner